src/exercises/Exercises.js

```diff
@@ -38,10 +46,12 @@ export default function Exercises({ api, articleID }) {
       api.getUserBookmarksToStudy(NUMBER_OF_EXERCISES, (bookmarks) => {
         setbookmarksToStudyList(bookmarks);
         NUMBER_OF_EXERCISES = bookmarks.length;
-        setCurretBookmarkToStudy(bookmarks[currentIndex]);
+        setCurrentBookmarkToStudy(bookmarks[currentIndex]);
+        api.wordsSimilarTo(bookmarks[currentIndex].id, (words) => {
```

Comment:
these three lines are also not needed if you move this behavior inside the MultipleChoice.js

Reply:
Removed those and a few more :-)